config/foreman.migrations/20200909151007_manage_acls_on_debian.rb

```diff
@@ -0,0 +1,3 @@
+if answers['foreman_proxy'].is_a?(Hash)
```

Comment:
I think you can check for Debian here to limit the scope
```suggestion
if answers['foreman_proxy'].is_a?(Hash) && facts[:os][:family] == 'Debian'
```

Untested though.